### PR TITLE
Add documentation for physcial infra provider topology widget

### PR DESCRIPTION
**Pull request by MaysaMacedo**
This PR contains documentation for the Topology widget.

### CODE REVIEW

File: physical_infra_provider_configuration/usage/topics/using_topology_widgets.adoc [hunk 1]
```diff
@@ -1 +1,21 @@
-=== Using the Topology Widget for Physical Infrastructure Providers
\ No newline at end of file
+=== Using the Topology Widget for Physical Infrastructure Providers
+
+**
+The Topology widget extends CloudForms overview capabilities for Physical Infrastructure providers relationships with Physical Servers, Host and VMs.
```
Comment (skovic) [inline physical_infra_provider_configuration/usage/topics/using_topology_widgets.adoc]: This sentence probably shouldn't be in bold.

File: physical_infra_provider_configuration/usage/topics/using_topology_widgets.adoc [hunk 2]
```diff
@@ -1 +1,21 @@
...
+
+**
+The Topology widget extends CloudForms overview capabilities for Physical Infrastructure providers relationships with Physical Servers, Host and VMs. 
+**  
+
+1.  Navigate to *Compute → Physical Infrastructure → Providers.*
+
+2.  Select the Physical Infrastructure provider to be detailed.
+
+3.  From the overview section, select Topology.
```
Comment (skovic) [inline physical_infra_provider_configuration/usage/topics/using_topology_widgets.adoc]: You should probably make Topology bold.

File: physical_infra_provider_configuration/usage/topics/using_topology_widgets.adoc (hunk 1 above)
Comment (MaysaMacedo) [reply physical_infra_provider_configuration/usage/topics/using_topology_widgets.adoc]: Thanks, I've just updated.

**Comment by MaysaMacedo** [comment]
@rodneyhbrown7 Thanks Rodney, I've just updated.

**Pull request merged**